src/App/Pages/Vault/VaultListGroupingsPage.cs

```diff
@@ -350,6 +350,10 @@ public GroupingOrCipherDataTemplateSelector(VaultListGroupingsPage page)
 
             protected override DataTemplate OnSelectTemplate(object item, BindableObject container)
             {
+                if (item == null)
```

Comment:
How could this occur?

Reply:
I see it with the UWP app, which has bugs with ListView and it ends up getting here. 

I also see it done here: https://blog.xamarin.com/customizing-list-view-cells-xamarin-forms-datatemplateselector/

Reply:
Weird. So what happens to the cell that ends up not getting a template?

Reply:
One of the UWP bugs results in your clicks being offset. So when you click on an element it triggers but not on an element (null). So nothing happens instead of a crash.